skimage/segmentation/slic_superpixels.py

```diff
@@ -88,12 +174,23 @@ def slic(image, n_segments=100, compactness=10., max_iter=10, sigma=0,
       interpret them as 3D with the last dimension having length 3, use
       `multichannel=False`.
 
+    * If spacing is None and mask is provided, chamfer distance
+      transform is used to homogeneously place seed points over the
+      mask, otherwise, the exact euclidian distance transform is used.
+
+    * `start_label` is introduced to handle the issue [4]_. The labels
+      indexing starting at 0 will be deprecated in future versions. If
+      `mask` is not `None` labels indexing starts at 1 and masked area
+      is set to 0.
+
     References
     ----------
     .. [1] Radhakrishna Achanta, Appu Shaji, Kevin Smith, Aurelien Lucchi,
         Pascal Fua, and Sabine Süsstrunk, SLIC Superpixels Compared to
         State-of-the-art Superpixel Methods, TPAMI, May 2012.
     .. [2] http://ivrg.epfl.ch/research/superpixels#SLICO
```

Comment:
```suggestion
    .. [2] https://www.epfl.ch/labs/ivrl/research/slic-superpixels/#SLICO
```
This link was no longer pointing to the expected info. I have updated it